### ISSUE TITLE
How to hide a particular spec field from the "Edit Form" UI

### ISSUE THREAD

**Issue by aharbis** [openshift/console#5321]
I tried to use the following:

```
// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=false
```

But the spec field is still being shown in the console UI when I click `Edit Form`. Is there a way to hide the field? I do not want to just stick it in `advanced`.

**Comment by aharbis** [comment]
Looking at the [OLM Descriptor Reference](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md) I do not see anything that looks like it would behave as "hidden"

**Comment by spadgett** [comment]
We've recently added a hidden spec description to 4.5, see #5094

@tlwu2013 @TheRealJon

**Comment by spadgett** [comment]
Closing since this should be fixed by #5094. Let us know if that doesn't address the issue.
